ArduCopter/Parameters.cpp

```diff
@@ -985,6 +985,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_GROUPINFO("AUTO_OPTIONS", 40, ParametersG2, auto_options, 0),
 #endif
 
+#if MODE_GUIDED_ENABLED == ENABLED
```

Comment:
I wonder if we should poke a set of parameters into the mode as we do for flowhold.

Should prefix with "GUID", I think - and generally the 4-character name we use for the modes.

Reply:
yes, you're probably right on both points.  I'll shorten the parameter name to GUID_OPTIONS.  I actually don't think that there will be any more parameters added for Guided mode although nobody knows the future of course.